docs/core/entity/valve.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Valve Entity
+sidebar_label: Valve
+---
+
+A valve entity controls an valve devices such as a the water or gas valves in your home. Derive a platform entity from [`homeassistant.components.valve.ValveEntity`](https://github.com/home-assistant/core/blob/dev/homeassistant/components/valve/__init__.py).
+
+## Properties
+
+:::tip
+Properties should always only return information from memory and not do I/O (like network requests). Implement `update()` or `async_update()` to fetch data.
+:::
+
+| Name | Type | Default | Description
+| ----------------------- | ---- | ------- | -----------
+| current_valve_position | <code>int &#124; None</code> | `None` | The current position of valve where 0 means closed and 100 is fully open.
+| is_closed | <code>bool &#124; None</code> | **Required** | If the valve is closed or not. Used to determine `state`.
```

Comment:
Let's add a sentence that `current_valve_position` will be used to calculate `is_closed` if `current_value_position` is set and `is_closed` is not set.